src/utils/connectors/redis.py

```
@@ -205,6 +206,13 @@ async def redis_log_streamer(
                 yield log
             except IndexError:  # No new line
                 await asyncio.sleep(1)  # Otherwise the stream will hang
+    except Exception as error:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)